webapp/App.js

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+/**
+ * Parent Component that is Rendered to the Page.
+ * @param {*} props arbituary input properties for our application to use
+ */
+export const App = (props) => {
+    return (
```

Comment:
consider using [tab sizes of 2 spaces](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation)

Reply:
207b340

Reply:
ditto

I believe you are using two [tabs](http://tarantsov.com/hackers-coding-style-guide/why-tabs-should-be-avoided.html#:~:text=This%20actually%20requires%20mixing%20tabs,like%20in%20the%20example%20above.) right now, not two spaces.

Reply:
Updated with commit 927ef05